lms/static/sass/views/_teams.scss

```diff
@@ -148,6 +149,200 @@
     @extend %t-weight4;
   }
 
+  .search-tools {
+    @extend %t-copy-sub1;
+    margin: ($baseline/10) $baseline;
+    color: $gray-l1;
+
+    p, span { // override
+      color: inherit;
+    }
+  }
+
+  .card {
+    @extend .clearfix;
+    position: relative;
+    display: inline-block;
+    width: 30%;
+    margin: ($baseline) 1.2%;
+    border: 1px solid $gray-l4;
+    border-bottom: 2px solid $gray-l3;
+    border-radius: 3px;
+    background-color: $gray-l5;
+    color: $black;
+
+    &:hover,
+    &:active,
+    &:focus {
+
+      .card-core-wrapper {
+        background-color: $white;
+      }
+
+      .card-meta-wrapper {
+        background-color: $gray-l5;
+      }
+    }
+
+    .card-core-wrapper {
+      @extend .clearfix;
+      transition: all .25s ease-in-out;
+      padding: ($baseline);
+    }
+
+    .card-title {
+      @extend %t-title5;
+      @extend %t-strong;
+      margin-bottom: .5em;
+    }
+
+    .card-description {
+      @extend %t-copy-sub1;
+      color: $gray;
+    }
+
+    .card-actions {
+
+      .action-view {
+        @extend %btn-pl-default-base;
+        float: right;
+        margin: ($baseline/4) 0;
+      }
+    }
+
+    .card-meta-wrapper {
+      @extend .clearfix;
+      @extend %t-copy-sub1;
+      transition: all .25s ease-in-out;
+      padding: ($baseline/2) $baseline;
+      background-color: $gray-l4;
+      color: $gray;
+
+      .meta-detail {
+        @extend %t-strong;
+        margin-top: ($baseline/4);
+        margin-right: ($baseline*.75);
+        color: $gray;
+
+        .icon {
+          margin-right: ($baseline/4);
+        }
+      }
+
+      p, span { // override
+        color: inherit;
+        font-weight: inherit;
+      }
+
+      .member-thumbs {
+        display: inline-block;
+        vertical-align: text-bottom;
+
+        img {
+          width: 20px;
+          height: 20px;
+          margin: 0 ($baseline/4);
+        }
+      }
+    }
+
+    &.square-card {
+
+      .card-description {
+        height: ($baseline*4);
```

Comment:
Yeah... because it is vertically cutoff the mixin and other css overflow methods won't work. I have this here as a safety but will be talking to the devs about cutting this off with js - or limiting the characters since there is no where you can actually see longer text.